### DISCUSSION TITLE
Tutorial for nvim-surround

### DISCUSSION THREAD

**Discussion by bcampolo** [kylechui/nvim-surround#291] [Show and tell]
Hi @kylechui, I was hoping to message you directly, but didn't see any contact info so hopefully this is OK.  I just wanted to say I really love the plugin and I just finished making a YouTube tutorial for nvim-surround and wanted to get your thoughts, and since I'm posting here, your community's thoughts as well.  The video can be found here: https://youtu.be/96FS45IaUgo

If you like it, feel free to use it on your GitHub or wherever you think is appropriate.  Either way, keep up the great work!

**Comment by kylechui** [comment]
Hey, thanks for the video! I really like how you're able to pack so much information into just 3 minutes. In particular, the "syntax highlighting" for the keypresses is a good detail that can help people make sense of the "verb" + "motion" format. If anything, I would say that perhaps for some of the rapid-fire sections you could go a bit slower to give people a chance to digest the mappings, but that's just a minor nitpick. I'll consider adding it to the README

**Reply by bcampolo** [reply]
Thanks for watching and for the great feedback!
